cppcache/src/Log.cpp

```diff
@@ -378,12 +421,21 @@ std::string Log::formatLogLine(LogLevel level) {
       now - std::chrono::system_clock::from_time_t(secs));
   auto tm_val = apache::geode::util::chrono::localtime(secs);
 
+  std::thread::id id = std::this_thread::get_id();
+  std::stringstream ss;
+  ss << id;
+  std::string threadIdWithName;
+  if (getThreadName() != "") {
+    threadIdWithName = ss.str() + " (" + getThreadName() + ")";
+  } else {
+    threadIdWithName = ss.str();
+  }
+
   msg << "[" << Log::levelToChars(level) << " "
       << std::put_time(&tm_val, "%Y/%m/%d %H:%M:%S") << '.' << std::setfill('0')
       << std::setw(6) << microseconds.count() << ' '
       << std::put_time(&tm_val, "%z  ") << g_hostName << ":"
-      << boost::this_process::get_id() << " " << std::this_thread::get_id()
-      << "] ";
+      << boost::this_process::get_id() << " " << threadIdWithName << "] ";
```

Comment:
just 
```c++
.. << std::this_thread::id << g_threadName << ...
```
Avoid the extra stream creation.

Reply:
Good catch. Done.